torch/_prims/__init__.py

```diff
@@ -1357,7 +1361,7 @@ def _collapse_view_aten(a: Tensor, start: int, end: int) -> Tensor:
   """
 
 collapse_view = _make_prim(
-    schema="collapse_view(Tensor(a) a, int start, int end) -> Tensor(a)",
+    schema="collapse_view(Tensor(a) a, SymInt[] original_shape, int start, int end) -> Tensor(a)",
```

Comment:
Information about the tensor's shape is lost in the pipeline, where we usually need only the number of dimensions and contiguity information.
Therefore, we need this additional argument to save the shape before view/reshape.

Reply:
We should have the metadata stored on `node.meta`, if you still have the FX graph around.

Reply:
Thank you for the suggestion! I'll try that.